src/main/java/htsjdk/variant/vcf/VCFFileReader.java

```diff
@@ -159,4 +156,8 @@ public void close() {
 			throw new TribbleException("Could not close a variant context feature reader.", ioe);
 		}
 	}
+
+    public boolean isQueriable() {
```

Comment:
javadoc

Reply:
And maybe can be used in the `query` method, to throw an informative exception if it cannot be queried..

Reply:
not sure what you mean in the second comment.

Reply:
That the query method catches IOE to throw a TribbleException. Now it can use this method to throw a TribbleException with a more informative error message (without changing the contract)